makefile

```diff
@@ -1,3 +1,21 @@
 default:
 	@cat makefile
-	
+
+synthetic_data:
+	wget https://gist.githubusercontent.com/Niarfe/25100a860cbb660150a3fec87378d8ac/raw/b280768fc7ad782c65f0e714017af2f05ba88ef2/synthetic_data.csv
+
```

Comment:
doh!  just noticed something... your job is named square_wave, (same comment for the fibonachi), but the file it creates is `square_wave.png`, please add the extension to the job name so the job won't repeat if the file exists!  Thanks!